bootloaders/riotboot/README.md

```diff
@@ -108,3 +98,43 @@ To test building, flashing and booting the first slot:
 For the second slot:
 
 `BOARD=samr21-xpro APP_VER=$(date +%s) make -C tests/riotboot/ riotboot/flash-slot1 test`
+
+# Quick riotboot porting guide
+
+Your board is not supported yet? Try to port riotboot!
+
+## Porting to a board based on am Arm Cortex-M0+/3/4/7 MCU
+
+Extending riotboot to support another board with a Cortex-M0+/3/4/7
+microcontroller is rather straightforward. You need to:
+
+  - Provide the variables `ROM_START_ADDR` and `ROM_LEN` as well as
+`RAM_LEN` and `RAM_START_ADDR`
+  - Adapt the linker scripts for your cpu to pass the test tests/cortexm_common_ldscript.
+To get an idea how to comply with cpu/cortexm_common/ldscrpts/cortexm.ld
+you might want to take a look how is it done for stm32 or sam0 CPU families.
+  - Make the startup code `board_init()` idempotent,
+i.e. whether you execute it once, or twice, it works all the same.
+  - Adapt the flasher script to support .bin flashing with an offset (if `make flash`
+integration is desired, and .bin flashing with offset is not supported yet). To get an idea, you can look at makefiles/tools/edbg.inc.mk
```

Comment:
As found again from the mailing list about cc2538dk, the flasher should also support partial updates and not erase the whole memory by default but just the required sections.